fake-clock.js

```diff
@@ -1888,9 +1888,9 @@ function withGlobal(_global) {
             typeof config === "number"
         ) {
             throw new TypeError(
```

Comment:
just ditch this whole thing?

Reply:
People have probably moved on by now, so that is most likely safe to do :)